Cargo.toml

```
@@ -19,7 +19,7 @@ include = [
 
 [dependencies]
 egui = "0.11.0"
-miniquad = { version = "=0.3.0-alpha.28" }
+miniquad = { version = "0.3.0-alpha.29" }
```

```suggestion
miniquad = { version = "=0.3.0-alpha.29" }
```

we should pin while using pre-releases as they can contain breaking changes